dydx_operations/views.py

```diff
@@ -26,8 +28,8 @@ def slow_withdrawal(self, request):
         data = serializer.data
         try:
             db_ref = OrderManager()
-            withdarwal_ref = DydxWithdrawal()
-            withdrawal_data = withdarwal_ref.slow_withdrawal(data)
+            withdrawal_ref = DydxWithdrawal()
```

Comment:
withdrawal_ref -> dydx_withdrawal_obj

Name class object variables the same as the ClassNames